perseo-main/src/main/java/com/telefonica/iot/perseo/LogLevelServlet.java

```diff
@@ -65,21 +66,28 @@
     @Override
     protected void doPut(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
-        String levelName = request.getParameter("level");
-        logger.info("changing log level to " + levelName);
-        Level level = levels.get(levelName);
-        if (level == null) {
-            logger.error("invalid log level: " + levelName);
-            response.setCharacterEncoding("UTF-8");
-            response.setContentType("application/json");
-            response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
-            response.getOutputStream().print("{\"errorMessage\":\"invalid log level\"}");
-            return;
+        try {
+        	String levelName = request.getParameter("level");
```

Comment:
Wrong indent

Reply:
Fixed in 05982ba4eefed043b201d8b95eb1c9b6190d2673